### DISCUSSION TITLE
Hide FormError if there is no error in the field whose name was passed in

### DISCUSSION THREAD

**Discussion by mule-stand** [ariakit/ariakit#3221] [General]
Is there a possibility to hide `FormError` if there is no error in the field whose name was passed? Because now it just shows an empty `div`, which causes problems with styling, for example when using gap, and in general is not very obvious. Or are there some accessibility constraints that require this element to be present?

**Comment by diegohaz** [comment]
The error element is a live region. Some screen readers need these live regions to be connected to the DOM before updating their contents, or else they won't announce them.

In the [Form docs](https://ariakit.org/components/form), we use this CSS to ensure the error element isn't affected by `gap`:

```css
.error:empty {
  display: none;
}
```
